pkg/ami/marketplace/scripts/1-per-boot.sh

```diff
@@ -24,15 +24,27 @@
 #
 ################################################################################
 
-sudo cp /etc/questdb/server.conf /var/lib/questdb/conf/
+if [ ! -f "/var/lib/questdb/conf/server.conf" ]; then
+  cp /etc/questdb/server.conf /var/lib/questdb/conf/server.conf
+else
+  cp /etc/questdb/server.conf /var/lib/questdb/conf/server.conf_new
```

Comment:
i think we can skip copying this file if it exists, we don't usually bring anything useful to user via that file